net/pfSense-pkg-frr/files/usr/local/pkg/frr.inc

```diff
@@ -674,10 +707,12 @@ function frr_generate_config_integrated() {
 		return;
 	}
 	if (empty($frr_conf['enable'])) {
-		/* FRR master switch is off. */
+		log_error("FRR Integrated: FRR master switch is off.");
 		return;
 	}
 
+	log_error("FRR Integrated: Rebuild configuration.");
```

Comment:
Same as above... Maybe wrap these in a debug setting that someone could opt into? Otherwise it seems way too chatty.